docs/getting-started-on-snowplow-open-source/faq/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Open Source quick start: frequently asked questions"
```

Comment:
Should this page URL be renamed to `getting-started-on-snowplow-community-edition` ?  Can we setup a redirect to not lose the current SEO for this page?